demosplan/DemosPlanCoreBundle/EventSubscriber/ProcedureMetricsEmailEventSubscriber.php

```diff
@@ -38,7 +37,7 @@ public function __construct(
         private readonly CustomerService $customerService,
         private readonly CurrentUserService $currentUser,
         private readonly MailService $mailService,
-        private readonly GlobalConfigInterface $globalConfig,
+        private readonly GlobalConfig $globalConfig,
```

Comment:
I would prefer to keep the Interface and the temporary warning as long as the method is not yet added to the interface. Or what was the reasoning behind this?

Reply:
That was the reason - changed in 43b6bf7bde25580984d2a42de5a4e2c84463c618

Reply:
also changed in planed pick in 176c41f5a52a94ab161f12fb7e17a00371674f78